Tests/eppo/PollerTests.swift

```diff
@@ -87,7 +87,7 @@ final class PollerTests: XCTestCase {
         try? await poller.start()
 
         // Buffer to account for the time it takes for the callback to execute
-        let buffer: UInt64 = 50_000_000
+        let buffer: UInt64 = 100_000_000
```

Comment:
Unrelated change. Flaky test that is passing more consistently if I increase the buffer